packages/dart_frog_cli/lib/src/daemon/domain/route_configuration_domain.dart

```diff
@@ -61,7 +61,7 @@ class RouteConfigurationDomain extends DomainBase {
         daemon.sendEvent(
           DaemonEvent(
             domain: domainName,
-            event: 'routeConfigurationChanged',
+            event: 'changed',
```

Comment:
Decided that this is a better name since route configuration is already in the domain

Reply:
Is it a breaking change?

Reply:
yes, of an unreleased feature lol